tools/test_api.py

```diff
@@ -2047,37 +2050,40 @@ def build_tests(tests, base_source_paths, build_path, target, toolchain_name,
         except Exception, e:
             if not isinstance(e, NotSupportedException):
                 result = False
-                
+
                 if continue_on_build_fail:
                     continue
                 else:
                     break
-        
+
         # If a clean build was carried out last time, disable it for the next build.
         # Otherwise the previously built test will be deleted.
         if clean:
             clean = False
-        
+
         # Normalize the path
         if bin_file:
             bin_file = os.path.normpath(bin_file)
-            
+
             test_build['tests'][test_name] = {
                 "binaries": [
                     {
                         "path": bin_file
                     }
                 ]
             }
-            
+
             print 'Image: %s'% bin_file
-    
+
     test_builds = {}
     test_builds["%s-%s" % (target.name, toolchain_name)] = test_build
-    
-    
+
+    # Print memory map summary on screen
+    print
+    print print_build_memory_usage_results(report)
```

Comment:
Can this be moved to `test.py` instead? All the reports are printed there at the moment, and that would allow you to toggle printing this report easier. See this line for an example of how the JUnit reports are being saved: https://github.com/mbedmicro/mbed/blob/master/tools/test.py#L197